hacking/test-module.py

```diff
@@ -89,6 +89,22 @@ def parse():
         return options, args
 
 
+def jsonify(result, format=False):
+    """ format JSON output (uncompressed or uncompressed) """
```

Comment:
```suggestion
    """Format JSON output (compressed or uncompressed) """
```

Reply:
This entire function was pre-existing code, it was just relocated without modification. The original is here:

https://github.com/ansible/ansible/blob/183c695be1312ac1d4829a990ab3cd5d1e4ca5a5/lib/ansible/parsing/utils/jsonify.py#L23